test/lib/completions/alias.exp

```diff
@@ -30,8 +30,8 @@ set cmd "alias foo"
 append cmd \002\002\002; # \002 = ^B = Move cursor left in bash emacs mode
 send "$cmd\t"
 expect {
-    -re "^alias foo\r\nbar +foo *\r\n/@alias foo$" { pass "$test" }
-    -re "^alias foo=$" { fail "$test: Wrong cursor position" }
+    -re "^alias foo(\b\b\b)?\r\nbar +foo *\r\n/@alias foo(\b\b\b)?$" { pass "$test" }
+    -re "^alias (foo\b\b\b)?foo=(foo\b\b\b)?$" { fail "$test: Wrong cursor position" }
```

Comment:
I think the difference depends on whether bash is compiled with or without curses.
 * If bash is compiled with curses then \b is *not* printed
 * If bash is compiled without curses then \b *is* printed
I've tested this to be true both with bash 4.3.30 and 4.4. Note that using `configure --without-curses` doesn't work (in the sense that the binary produced is still linked against ncurses) so I had to remove `ncurses-devel` to produce the non-curses binaries.

```
$ cat a.exp 
set env(PS1) "/@"
set env(TERM) "dummy"
spawn bash --norc
expect "/@"
send "echo hola\002\002\002\n"
expect "hola\r\n"
$ PATH=~/src/bash/bash-4.4-nocurses/ /usr/bin/expect a.exp | grep "echo hola" | od -c
0000000   /   @   e   c   h   o       h   o   l   a  \b  \b  \b  \r  \n
0000020
$ PATH=~/src/bash/bash-4.4-curses/ /usr/bin/expect a.exp | grep "echo hola" | od -c
0000000   /   @   e   c   h   o       h   o   l   a  \r  \n
0000015
$ PATH=~/src/bash/bash-4.3.30-nocurses/ /usr/bin/expect a.exp | grep "echo hola" | od -c
0000000   /   @   e   c   h   o       h   o   l   a  \b  \b  \b  \r  \n
0000020
$ PATH=~/src/bash/bash-4.3.30-curses/ /usr/bin/expect a.exp | grep "echo hola" | od -c
0000000   /   @   e   c   h   o       h   o   l   a  \r  \n
0000015
```

Note that if one does *not* set `TERM=dummy`, then \b are printed in both cases.